test-runtime/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "test-runtime"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+subxt = { path = ".." }
+sp-runtime = { package = "sp-runtime", git = "https://github.com/paritytech/substrate/", branch = "master" }
+codec = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive", "full", "bit-vec"] }
+
+[build-dependencies]
+hex = "0.4.3"
+reqwest = { version = "0.11.6", features = ["blocking", "json"] }
```

Comment:
Since we already depend on `subxt` above we could avoid this extra (another!) http client dependency. Should be able to use `subxt::RpcClient`

Reply:
Good point! I'll make use of that or one of the other http deps instead!

Reply:
> Also, I remember someone in my presentation (@athei or @dvdplm?) suggesting we could even enable the option for the macro to fetch the metadata from the running substrate node, which may be something we can do in the future for this use case.

Yeah this was my suggestion. To download the metadata inside the proc macro.

Reply:
I think that's best left for a separate PR if we want it. It does strike me that adding the feature to the macro would only really be beneficial for this sort of testing though, so I actually quite like the idea that it's separate and that the macro doesn't become a little more complex to handle it :)